packages/blueprints-integration/src/context/partsAndPieceActionContext.ts

```diff
@@ -48,6 +49,12 @@ export interface IPartAndPieceActionContext {
 	/** Gets the Segment. This primarily allows for accessing metadata */
 	getSegment(segment: 'current' | 'next'): Promise<IBlueprintSegment | undefined>
 
+	/** Get a list of the upcoming Parts in the Rundown, in the order that they will be Taken */
+	getUpcomingParts(
+		/** The max number of parts returned. */
+		limit?: number
```

Comment:
I like that the implementation of this interface has a reasonable default of `5`, but I think the fact that there is a default has to be documented here (maybe not necessarily what the default is, but that there is a default?).

Alternatively, just mandating that you need to provide your own limit would be fine too, I think.